applications/MeshMovingApplication/custom_elements/laplacian_meshmoving_element.h

```diff
@@ -163,27 +163,21 @@ class LaplacianMeshMovingElement : public Element {
   ///@{
   ///@}
 
-  ///@name Serialization
-  ///@{
-  friend class Serializer;
-  LaplacianMeshMovingElement() {}
-  ///@}
-
   ///@name Private Operators
   ///@{
   ///@}
 
   ///@name Private Operations
   ///@{
 
+  // A private default constructor necessary for serialization
+  LaplacianMeshMovingElement() {}
+
   void CalculateDeltaPosition(VectorType &IntermediateDisplacements,
-                              ProcessInfo &rCurrentProcessInfo);
+                              const ProcessInfo &rCurrentProcessInfo) const;
 
   void CheckElementMatrixDimension(MatrixType &rLeftHandSideMatrix,
-                                   VectorType &rRightHandSideVector);
-
-  MatrixType CalculateDerivatives(const int &rdimension,
```

Comment:
@AndreasWinterstein this function is not implemented...

Reply:
Your are right. This was necessary before and is now directly obtained from the geometry. So it is correct to delete it.